back/internal/dbg/dbg.go

```diff
@@ -25,27 +25,27 @@ func new() *dbg {
 }
 
 func (d *dbg) logln(as ...interface{}) {
-	debug.log.Println(as...)
+	toggle := debug.atomVal.Load().(bool)
```

Comment:
Not related to this line, but rather line 18. Move `var debug = new()` closer to the exported functions as it's more related to those than the dbg package type and methods.

Reply:
related to this line, use d.atomVal.Load()... instead of debug.atomVal. per discussion